### ISSUE TITLE
The format of the task list is not the same as github  

### ISSUE THREAD

**Issue by yama-syo** [markedjs/marked#1981]
**What pain point are you perceiving?.**

The format of the task list is not the same as github  
```
+  [x] (+, 2space, [x], space)
+   [x] (+, 3space, [x], space)
```
↑ Can be a tasklist?
result
* OK: github  
* NG: marked

To Reproduce  

github  
+  [x] tasklist1
+   [x] tasklist2

Marked(version: master)
[Marked Demo](https://marked.js.org/demo/?text=%23%23%20tasklist%0A%23%23%23%20NG(github%20OK)%0A%2B%20%20%5Bx%5D%20NG(%2B%2C2space%2C%5B%2Cx%2C%5D%2Cspace%2CN%2CG)%0A%2B%20%20%20%5Bx%5D%20NG(%2B%2C3space%2C%5B%2Cx%2C%5D%2Cspace%2CN%2CG)%0A&options=%7B%0A%20%22baseUrl%22%3A%20null%2C%0A%20%22breaks%22%3A%20false%2C%0A%20%22gfm%22%3A%20true%2C%0A%20%22headerIds%22%3A%20true%2C%0A%20%22headerPrefix%22%3A%20%22%22%2C%0A%20%22highlight%22%3A%20null%2C%0A%20%22langPrefix%22%3A%20%22language-%22%2C%0A%20%22mangle%22%3A%20true%2C%0A%20%22pedantic%22%3A%20false%2C%0A%20%22sanitize%22%3A%20false%2C%0A%20%22sanitizer%22%3A%20null%2C%0A%20%22silent%22%3A%20false%2C%0A%20%22smartLists%22%3A%20false%2C%0A%20%22smartypants%22%3A%20false%2C%0A%20%22tokenizer%22%3A%20null%2C%0A%20%22walkTokens%22%3A%20null%2C%0A%20%22xhtml%22%3A%20false%0A%7D&version=master "Marked Demo")
  

github recognizes a list as a task list if there is one or more space between the list symbol and [x].    
marked recognizes a list as a task list if there is only one space between the list symbol and the [x].  

**Describe the solution you'd like**  
So, I'd like the format of task lists to be the same as github, with one or more space between the list symbol and [x].

**Comment by UziTech** [comment]
PRs are appreciated 😁👍

**Comment by calculuschild** [comment]
This should be fixed by #2112
